x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/loadingservice/LocalModel.java

```diff
@@ -56,6 +56,7 @@
     private final License.OperationMode licenseLevel;
     private final CircuitBreaker trainedModelCircuitBreaker;
     private final AtomicLong referenceCount;
+    private final long modelSize;
```

Comment:
maybe rename to `cachedRamBytesUsed` and add a comment to remember why it's a good idea to cache it?